shardingsphere-control-panel/shardingsphere-orchestration/shardingsphere-orchestration-core/shardingsphere-orchestration-core-facade/src/main/java/org/apache/shardingsphere/orchestration/core/facade/ShardingOrchestrationFacade.java

```diff
@@ -81,7 +81,7 @@
     
     private final ShardingOrchestrationListenerManager listenerManager;
     
-    public ShardingOrchestrationFacade(final OrchestrationConfiguration orchestrationConfig, final Collection<String> shardingSchemaNames) {
+    public ShardingOrchestrationFacade(final OrchestrationConfiguration orchestrationConfig, final Collection<String> shardingSchemaNames, final int identifier) {
```

Comment:
The parameter `identifier` cannot correctly describe the meaning of orchestration instance unique identification,  my suggestion is `instanceId`, and i think it's better to define it as `String` type.

Reply:
keeping this as `instanceId` also does not seem to be justified since this is only a part of the actual `instanceId`.
Should we keep this as `instanceSubId`?

Reply:
> keeping this as `instanceId` also does not seem to be justified since this is only a part of the actual `instanceId`.
> Should we keep this as `instanceSubId`?

How about `instanceSign` or `instanceTag`, my understanding, this parameter is just a sign or tag of `instanceId`.

Reply:
`instanceTag` sounds good!

Reply:
Looking forward to your submission for optimization.